client/js/lounge.js

```diff
@@ -430,6 +430,14 @@ $(function() {
 		}
 	});
 
+	chat.on("click", ".condensed", function() {
+		$(this).toggleClass("closed");
+	});
+
+	chat.on("click", ".condensed div", function(e) {
```

Comment:
Pretty sure you can just do `e.target !== e.currentTarget` in the event above instead of having this extra event.